src/hooks/useMember/useMember.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import {
+  getMember as getMemberFn,
+  addMember as addMemberFn,
+  setActiveChannel as setActiveChannelFn,
+  removeMember as removeMemberFn,
+  addChannel as addChannelFn,
+} from './utils'
+
+export type Member = {
+  id: string
+  handle: string
+  avatarUrl: string
+  channels: Channel[]
+  activeChannel: Channel
```

Comment:
I'm not sure if we actually should save anything except the IDs. It will probably make more sense to just fetch all the rest. Let's discuss in the call